app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -159,4 +165,55 @@ public function getStoreIds()
         return $storeIds;
     }
 
+    private function getOrderProducts(Mage_Sales_Model_Order $order): array {
+        $products = [];
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            if ($product->isConfigurable()) {
+                continue;
+            }
+            $products[] = [
+                'name' => $product->getName(),
+                'url' => $this->getProductUrl($product),
+                'id' => $product->getId(),
+                'sku' => $product->getSku(),
+                'image_url' => $this->getProductImageUrl($product),
+                'brand' => $product->getAttributeText('manufacturer'),
+            ];
+        }
+
+        return $products;
+    }
+
+    private function getProductImageUrl(Mage_Catalog_Model_Product $product): string {
+        if ($product->getMediaGalleryImages()->getSize() < 1) {
+            $parent_id = Mage::getModel('catalog/product_type_configurable')->getParentIdsByChild($product->getId())[0];
+            if (!empty($parent_id)) {
+                $parent_product = Mage::getModel('catalog/product')->load($parent_id);
+                return $parent_product->getImageUrl();
+            }
+        }
+        return $product->getImageUrl();
+    }
+
+    private function getLanguage(int $storeId, Mage_Sales_Model_Order $order): string {
+        $language = Mage::getStoreConfig('webwinkelconnect/invitation/language', $storeId);
+        if (empty($language)) {
```

Comment:
Use `empty` only if you also need to check for something being defined.

```suggestion
        if (!$language) {
```